diff -Nur a/src/module.c b/src/module.c
--- a/src/module.c	2013-04-26 16:49:03.022174178 +0100
+++ b/src/module.c	2016-10-22 18:41:42.151021720 +0100
@@ -96,7 +96,10 @@
     int retries = 30;
     bb_log(LOG_INFO, "Unloading %s driver\n", driver);
     char *mod_argv[] = {
-      "rmmod",
+      "modprobe",
+      "-r",
+      "nvidia_modeset",
+      "nvidia_drm",
       driver,
       NULL
     };
