diff -Nur a/epkg b/epkg
--- a/epkg	2015-01-09 17:30:57.000000000 +0000
+++ b/epkg	2016-04-06 17:31:27.710560138 +0100
@@ -6,7 +6,7 @@
 case "$action" in
 	# "Standard" action commands, used regularly
 	search | -s)
-		eix "$@"
+		emerge -s "$@"
 		;;
 	install | -i)
 		emerge -av "$@"
@@ -15,7 +15,7 @@
 		emerge --depclean -av "$@"
 		;;
 	update)
-		eix-sync "$@"
+		kogaionsync
 		;;
 	upgrade | dist-upgrade)
 		# Make it portage 2.1-safe and make sure it catches EVERYTHING.
@@ -41,19 +41,19 @@
 		;;
 	# info commands
 	listfiles)
-		equery files "$@"
+		qlist "$@"
 		;;
 	listinstalled)
-		equery list '*' "$@"
+		qlist -ICv "$@"
 		;;
 	provides)
-		equery belongs "$@"
+		qfile "$@"
 		;;
 	verify)
-		equery check -o "$@"
+		qcheck "$@"
 		;;
 	verifyall)
-		equery check -o '*' "$@"
+		qcheck
 		;;
 	# random commands
 	moo)
@@ -85,11 +85,11 @@
 			    autoclean - Remove sources for packages no longer installed [eclean-dist -df]
 			    clean - Remove *ALL* package sources [rm -rf $DISTDIR/*]
 			    forceremove - *Unsafely* remove packages [emerge --unmerge -av package(s)]
-			    listfiles - List the files belonging to a package [equery files package]
-			    listinstalled - List installed packages [equery list '*']
-			    provides - List the installed package(s) which own the indicated file [equery belongs file]
-			    verify - Verify a package's installed files match the checksum and timestamp they had when first installed [equery check -o package]
-			    verifyall - Same as above, but against all installed packages [equery check -o '*']
+			    listfiles - List the files belonging to a package [qlist package]
+			    listinstalled - List installed packages [qlist -ICv package]
+			    provides - List the installed package(s) which own the indicated file [qfile file]
+			    verify - Verify a package's installed files match the checksum and timestamp they had when first installed [qcheck package]
+			    verifyall - Same as above, but against all installed packages [qcheck]
 			    sysinfo - Display information about installed core packages and portage configuration [emerge --info]
 
 			You can pass arbitrary parameters to us after the command you want, and we'll
