diff -Nur a/src/modules/packages/main.py b/src/modules/packages/main.py
--- a/src/modules/packages/main.py	2016-04-12 10:23:47.000000000 +0100
+++ b/src/modules/packages/main.py	2016-08-12 16:17:12.660215490 +0100
@@ -87,6 +87,7 @@
             check_target_env_call(["pacman", "-Rs", "--noconfirm"] + pkgs)
         elif self.backend == "portage":
             check_target_env_call(["emerge", "-C"] + pkgs)
+            check_target_env_call(["emerge", "--depclean", "-q"])
         elif self.backend == "entropy":
             check_target_env_call(["equo", "rm"] + pkgs)
 
